uniffi_bindgen/src/bindings/kotlin/templates/Float32Helper.kt

```diff
@@ -1,4 +1,4 @@
-internal object FfiConverterFloat: FfiConverter<Float, Float> {
+public object FfiConverterFloat: FfiConverter<Float, Float> {
```

Comment:
I'm mildly surprised you need to make all `FfiConverter` based objects public, but I assume that's necessary due to Kotlin requirements.

Reply:
We're generating a different package for each UDL file, but with external types one package needs to import from the other so I think these need to be public.  I'll add a comment about this.